storage/innobase/xtrabackup/src/xbcloud/xbcloud.cc

```diff
@@ -1135,7 +1135,7 @@ int main(int argc, char **argv) {
         storage_class,
         opt_google_endpoint != nullptr ? opt_google_endpoint
                                        : "https://storage.googleapis.com/",
-        LOOKUP_DNS, S3_V2));
+        LOOKUP_DNS, S3_V4));
```

Comment:
what is the implication of this? will older versions not work?

Reply:
it is signing process. I am not sure why we use V2 with GC and not V4.  GC supports V4 and it recommends users to use that.

Reply:
my concern is about newer xbcloud versions not being able to restore older backups stored in GC. Will there be any issues?
Can you upload a backup without your fix and restore using xbcloud using V4?

Reply:
yes. upload with V2 without my patch. and download with V4 with my patch is working fine.

Reply:
Ok